translate/src/lib.rs

```diff
@@ -0,0 +1,86 @@
+pub mod cli;
+pub mod runner;
+pub mod scheduler;
+pub mod tools;
```

Comment:
I only made so many modules because I wanted to split the source code into several files. I don't think we should publicly export the `cli`, `runner`, or `scheduler` modules, instead I think we should `pub use` the specific types that we want to make public.

Then users of the library will see a flat hierarchy except for `tools`.